backend/apps/schemes/api/views/scheme_period_views.py

```
@@ -0,0 +1,145 @@
+    ordering = ["-period_number"]
+
+    def create(self, request, *args, **kwargs):
+        """Create is not allowed - use Scheme creation or renewal instead."""
+        return Response(
+            {
+                "error": "Use /api/schemes/{id}/renew/ to create new periods or create scheme with initial period"
+            },
+            status=status.HTTP_405_METHOD_NOT_ALLOWED,
+        )
+
+    def update(self, request, *args, **kwargs):
```

Ruff (0.14.3)

Unused method argument: `args`

(ARG002)